week-1/mandatory/2-classes-db/task.md

```diff
@@ -7,7 +7,49 @@ Below you will find a set of tasks for you to complete to consolidate and extend
 To submit this homework write the correct commands for each question here:
 
 ```sql
+1. select rate from rooms where rate > 100;
+   select *  from rooms where rate > 100;
```

Comment:
Very good - the first query doesn't identify which rooms meet the requirement but the second does.